### ISSUE TITLE
Does this polyfill works for chrome version 38 ?

### ISSUE THREAD

**Issue by gaganhn** [tvcutsem/harmony-reflect#84]
I see there are errors for reflect.js in my console.
<img width="1727" alt="Screenshot 2023-07-21 at 3 41 10 PM" src="https://github.com/tvcutsem/harmony-reflect/assets/26406858/5f88307f-8eda-4ba4-966a-e36cf716390d">

**Comment by tvcutsem** [comment]
Yes, this polyfill should work on chrome v38, but as noted in the readme:

> Chrome (>= v19 && <= v37) used to support proxies behind a flag (chrome://flags/#enable-javascript-harmony) but Chrome v38 [removed](https://code.google.com/p/v8/issues/detail?id=1543#c44) the Proxy constructor. As a result, this library cannot patch the harmony-era Proxy object on Chrome v38 or above.

Can you post more diagnostics on the error (including a full stack trace)? Does the error appear only on Chrome v38 or also on newer/latest releases?

**Comment by tvcutsem** [comment]
To clarify, this library cannot properly emulate ES6+ `Proxy` objects on Chrome v38, but can support the `Reflect` API.

**Comment by gaganhn** [comment]
Oh,
I mainly wanted this polyfill for Proxy.

Thanks for reply!
